lib/slime/parser/embedded_engine.ex

```diff
@@ -17,19 +16,9 @@ defmodule Slime.Parser.EmbeddedEngine do
   }
   |> Map.merge(Application.get_env(:slime, :embedded_engines, %{}))
   |> Enum.into(%{}, fn ({key, value}) -> {to_string(key), value} end)
-  @registered_engines Map.keys(@engines)
-
-  def parse(header, lines) do
-    case Regex.named_captures(@embedded_engine_regex, header) do
-      %{"engine" => engine, "indent" => indent} when engine in @registered_engines ->
-        indent = String.length(indent)
-        {embedded_lines, rest} = split_lines(lines, indent)
-        {{indent, render_with_engine(engine, embedded_lines)}, rest}
-      _ -> nil
-    end
-  end
 
-  def render_with_engine(engine, lines) when is_list(lines) do
+  def render_with_engine(engine, line_contents) when is_list(line_contents) do
+    lines = Enum.map(line_contents, &compile/1)
```

Comment:
Thanks! I didn't think about that; it certainly makes sense to decide on how to render interpolation on a per engine basis, as your example shows. I've added it as a test case.

On a somewhat unrelated note, this should probably be mentioned in the release notes — it seems to me the [Markdown example](https://github.com/slime-lang/slime#embedded-engines) won't work anymore.